build.sbt

```diff
@@ -11,8 +11,7 @@ def previousVersion = "0.7.0"
 def scala213 = "2.13.2"
 def scala212 = "2.12.11"
 def scala211 = "2.11.12"
-def dottyNext = "0.27.0-RC1"
-def dottyStable = "0.26.0"
+def dottyStable = "3.0.0-M1"
```

Comment:
how about renaming this to scala3?

Reply:
Changed it!

Reply:
And remembered the env variables too in the next push :sweat_smile: